game-core/src/main/java/games/strategy/engine/data/MutableProperty.java

```diff
@@ -145,6 +146,20 @@ public void resetValue() {
     return ofString(stringSetter, noGetter(), noResetter());
   }
 
+  public static <T> MutableProperty<T> ofMapper(
```

Comment:
Even though we're going to shut off the Checkstyle JavadocMethod rule, I'd document this a bit because it wasn't clear to me at first exactly what "mapper" meant.  Honestly, I think "parser" is a better term, but I'd leave it as-is because I'm thinking there may be a future refactoring where we use the term "parser" instead of "string setter", and this would just get pulled into that.